src/client/binance_client.py

```diff
@@ -254,36 +254,58 @@ def position_summary(self, symbol_list=None) -> pd.DataFrame:
                 trade_history = self.client.my_trades(symbol=symbol)
                 live_px = float(self.client.ticker_price(symbol=symbol)['price'])
                 symbol_data = []
+                test_df = pd.DataFrame(trade_history)
                 for trade_info in trade_history:
                     coin_data = {}
                     coin_data['Symbol'] = trade_info['symbol']
-                    coin_data['QTY'] = float(trade_info['qty']) if trade_info['isBuyer'] == True else float(
-                        trade_info['qty']) * -1
-                    coin_data['WAP'] = float(trade_info['price']) * float(trade_info['qty'])
-                    coin_data['FEE'] = float(trade_info['commission'])
-                    coin_data['Side'] = 'Buy' if trade_info['isBuyer'] == True else 'Sell'  # 1 is buy and 0 Sell
-                    coin_data['PnL'] = ((live_px - float(trade_info['price'])) * float(coin_data['QTY'])) - float(
-                        trade_info['commission'])
+                    coin_data['FEE'] = float(
+                        trade_info['commission'])  # Commision for sells are in GBP and Coin for Buys
+                    coin_data['live_px'] = live_px
+                    coin_data['trade_price'] = trade_info['price']
+
+                    if trade_info['isBuyer'] == True:  # Trade is a buy
+                        coin_data['Side'] = 'Buy'
+                        coin_data['QTY'] = float(trade_info['qty']) - float(
+                            trade_info['commission'])  # Remove commission from buy
+                        coin_data['WAP'] = float(trade_info['price']) * coin_data[
+                            'QTY']  # Create weight for working out the weighted average buy price
+                        coin_data['PnL'] = ((live_px - float(trade_info['price'])) * float(coin_data['QTY'])) - float(
+                            trade_info['commission']) * float(trade_info['price'])
+                        coin_data['FEE'] = float(trade_info['commission']) * float(
+                            trade_info['price'])  # Commision for sells are in GBP and Coin for Buys
+                        # Don't include the commission as it's already removed from the qty
+
+                    else:  # Trade is a Sell
```

Comment:
Just so we can come at a glance and remember how it works, can you add a docstring to the function and to the Buy/Sell cases. Just a quick few sentences like this:

Buying:
1. QTY purchased is computed by taking the purchased quantity minus the quantity in crypto
2. WAP using that quantity 
3. etc etc